agent-identity/python/ciba-on-behalf-of-flow/sleeping-guardian/market_engine.py

```
@@ -0,0 +1,191 @@
+                f"{self.mcp_server_url}/api/market/update",
+                json={"symbol": self.symbol, "new_price": new_price},
+                timeout=2
+            )
+            if response.ok:
+                self.last_synced_price = new_price
+                print(f"[Market Engine] ✓ Synced price ${new_price:.2f} to MCP server")
+            else:
+                print(f"[Market Engine] ⚠️  Failed to sync price to MCP: {response.status_code}")
+        except Exception as e:
+            # Don't crash if MCP sync fails - just log it
+            pass  # Silently ignore sync errors to avoid spam
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)